recipe/recipe.yaml

```diff
@@ -0,0 +1,79 @@
+context:
+  name: qdldl-python
+  version: 0.1.7.post5
+  lib_ver: ${{ (version | split("."))[:3] | join(".") }}
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/osqp/${{ name }}/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 498002a3826dca9d03186972ca82ee8acb930127ea5603b94a845327ffa98a7c
+  patches:
+    - patches/0001-use-Ninja-everywhere.patch
+    - patches/0002-build-against-conda-forge-s-libqdldl.patch
+    - patches/0003-build-qdldlamd-as-shared-lib-and-install-it.patch
+    - patches/0004-Do-not-pass-CMAKE_GENERATOR-as-variable.patch
+
+build:
+  number: 1
+  script:
+    - python -m pip install . -vv
+
+requirements:
+  build:
+    - if: build_platform != target_platform
+      then:
+        - python
+        - cross-python_${{ target_platform }}
+        - numpy
+    - ${{ compiler('c') }}
+    - ${{ stdlib("c") }}
+    - ${{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - python
+    - pip
+    - setuptools
+    - libqdldl =${{ lib_ver }}
+    - numpy
+    - pybind11
+  run:
+    - python
+    - scipy
+
+tests:
+  - script:
```

Comment:
Reorder these sections:
First the python import tests, then the requirements for running the test suite, then the files for that, then the actual invocation

Reply:
Import tests have been moved.

AFAICT, the list of commands needs to be the first child of the `script` element.